log4j-api/src/main/java/org/apache/logging/log4j/message/ReusableObjectMessage.java

```diff
@@ -128,7 +128,9 @@ public <S> void forEachParameter(final ParameterConsumer<S> action, final S stat
 
     @Override
     public Message memento() {
-        return new ObjectMessage(obj);
+        Message message = new ObjectMessage(obj);
+        message.getFormattedMessage();
```

Comment:
I presume you make this call to trigger the stack trace capture. Would you mind documenting this, please? (Also in other `message/*Message.java` changes.)

Reply:
Fixed in https://github.com/apache/logging-log4j2/pull/3171/commits/6812e5e3c4cb7181ebf9dd62f0c17e33ef07af2d